File: 2260-divide-a-string-into-groups-of-size-k/divide-a-string-into-groups-of-size-k.py
class Solution:
    def divideString(self, s: str, k: int, fill: str) -> List[str]:
        res =[]
        if len(s)%k !=0:
            missing=k-(len(s) % k)
            s+=fill*missing
        for i  in range(0,len(s),k):
            res.append(s[i:i+k])
        return res

File: 2260-divide-a-string-into-groups-of-size-k/divide-a-string-into-groups-of-size-k.cpp
class Solution {
public:
    vector<string> divideString(string s, int k, char fill) {
        int n=s.size();
        int mod=n%k;
        if(mod!=0)
        {
            s.append(k-mod,fill);
        }
        
        vector<string>res;
        n=s.size();
        for(int i=0;i<n;i+=k)
        {
            res.push_back(s.substr(i,k));
        }
        return res;
    }
};